.releaserc.json

```diff
@@ -40,12 +44,7 @@
         }
       }
     ],
-    [
-      "@semantic-release/npm",
-      {
-        "npmPublish": false
-      }
-    ],
+    "@semantic-release/npm",
```

Comment:
Not entirely sure, but I think that we need at least the following block to the `package.json` to make this working as expected:

```json
"files": [
  "dist"
]
```

`dist` might be adjusted to the actual distribution directory.